File: spec/code_samples/Node/exchange-keys@{keyId}/parameters.js
var http = require("http");

var options = {
  "method": "PARAMETERS",
  "hostname": "ec2-3-222-208-106.compute-1.amazonaws.com",
  "port": "8086",
  "path": "/exchange-keys/{keyId}",
  "headers": {}
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();

File: spec/code_samples/Python/exchange-keys@{keyId}@activate/parameters.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

conn.request("PARAMETERS", "/exchange-keys/{keyId}/activate")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))